test/burials/config/documents.unit.spec.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-dom/test-utils';
+
+import { DefinitionTester, getFormDOM } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/burials/config/form';
+
+describe('Burials document upload', () => {
+  const { schema, uiSchema, depends } = formConfig.chapters.additionalInformation.pages.documentUpload;
+
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = getFormDOM(form);
+
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(0);
```

Comment:
I mainly wanted to make sure nothing showed up if you didn't make the right choices for form data. Normally you won't hit this state because of `depends`.